app/uk/gov/hmrc/agentservicesaccount/views/pages/manage_account.scala.html

```diff
@@ -54,7 +54,7 @@ <h3 class="govuk-heading-s" id="status-key">
                 @if(config.accessGroups.size > 1) {
                     @ul_html(items = config.accessGroups.map { link =>
                         if (link.renderAsButton) {
-                            link_as_button(key = link.msgKey, href = link.href)
+                            link_as_button(key = link.msgKey, href = "http://localhost:9452/agent-permissions/create-group/select-group-type")
                         } else {
                             a(key = link.msgKey, href = link.href)
```

Comment:
the href needs changing in more than one place, in the else case it would still go to the old one. I recommend going to `ManageAccessPermissionsConfig` instead of changing it in each of these places though.